### DISCUSSION TITLE
Why is Q4_0_4_4 support removed?

### DISCUSSION THREAD

**Discussion by kjh2159** [ggml-org/llama.cpp#10847] [Q&A]
I could load a Q4_0_4_4 quantized model before. However, after pulling git repository, I couldn't load Q4_0_4_4 models and cli throws a message, "Q4_0_4_4 is removed, use Q4 instead".

Does anyone know about the reason why Q4_0_4_4 support is removed?

**Comment by DavidZyy** [comment]
I am also curious about this question. In the previous version that the codes is not deleted, I saw the comment about these codes are "activation aware quantization", but it seems have some differences with the method that [this paper](https://arxiv.org/abs/2306.00978) described.

**Comment by ExtReMLapin** [comment]
Next time you go find the code part [where it say it's removed and look at the git blame](https://github.com/ggerganov/llama.cpp/blame/7b1ec53f56bb72c49e4c8434157895f94d3709c2/ggml/src/ggml.c#L795)

You'll find this [PR](https://github.com/ggerganov/llama.cpp/pull/10446) with the reasons

**Reply by DavidZyy** [reply]
Thank you. It is very helpful.

**Comment by kjh2159** [comment]
Thank you for answering. I figured out the reason why.
